uri/README.md

```diff
@@ -0,0 +1,41 @@
+URI specifications
+==================
+
+Authors: [Patrik Wallström](github.com/pawal)
+
+Reviewers:
+
+* * *
+
+# Abstract
+
+For different applications to handle ipfs references there need to be URIs registered. In order to for applications to understand how to handle the references, the URIs needs to have specificaitions. The generic syntax for URIs are described in [RFC3986](https://tools.ietf.org/html/rfc3986).
+
+The consensus of which URIs need to be specified can be found here: https://github.com/ipfs/go-ipfs/issues/1678#issuecomment-157478515
+
+# Status of this spec
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+Note: These specifications are currently in the wip phase, things are likely to change.
+
+# Overview
+
+The discussion on what URI schemes that are needed is concluded [here](https://github.com/ipfs/go-ipfs/issues/1678#issuecomment-157478515). The discussion lead to the consensus that these URI schemas are needed:
+
+ * **canonical NURI** is a file system **path**. Does not need a URI specification. Examples, where **Qm...** is a [multihash](https://github.com/multiformats/multihash):
+    * /ipfs/Qm...
+    * /ipns/Qm...
+    * /ipfs/Qm.../filename
+    * /ipns/Qm.../dir/filename
+
+ * **[ipfs://](./ipfs.md)** style URI that references IPFS content with the multihash and filepath.
+
+ * **[fs://](./fs.md)** style URI referencing ipfs-specific AND non-ipfs specific hash resolution mechanisms
```

Comment:
According to https://github.com/ipfs/specs/pull/152#issuecomment-284628862 `fs://` URL is replaced by `dweb:` URI